client_folder_react_app/ethisim/src/App.js

```diff
@@ -2,26 +2,20 @@ import React from 'react';
 import './App.css';
 import {
   BrowserRouter as Router,
-  Switch,
   Route,
   Redirect,
 } from "react-router-dom";
+import Signup from './pages/signup';
 import Login from './pages/login';
-
 export default function App() {
   return (
     <Router>
         <Route exact path="/">
           <Redirect to="/login" />
         </Route>
 
-        <Switch>
-          <Route path="/login">
-            <Login />
-          </Route>
-        </Switch>
+        <Route path = "/login" component={Login}/>
+        <Route path="/signup" component={Signup}></Route>
```

Comment:
Should keep style consistency, and either explicitly include the </Route> on both, or neither